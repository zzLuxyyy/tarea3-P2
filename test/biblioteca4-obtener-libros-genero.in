crearBiblioteca
agregarGeneroABiblioteca -1 50 Accion
agregarGeneroABiblioteca 50 18 Policial
agregarGeneroABiblioteca 18 17 Noir
agregarGeneroABiblioteca 17 100 Noir moderno
agregarGeneroABiblioteca 18 13 Moderno
agregarGeneroABiblioteca 50 22 Aventura
crearFecha 01/01/1949
crearLibro 1984 "1984" "George" "Orwell" "Distopía" 50
agregarLibroBiblioteca
crearFecha 01/01/1932
crearLibro 1932 "Un mundo feliz" "Aldous" "Huxley" "Otra distopía" 22
agregarLibroBiblioteca
crearFecha 02/02/1960
crearLibro 1933 "El Amor en los tiempos de colera" "Gabriel" "Garcia Marquez" "Romantico" 18
agregarLibroBiblioteca
crearFecha 03/03/1961
crearLibro 1934 "Harry Potter" "JK" "Rowling" "Aventura" 18
agregarLibroBiblioteca
crearFecha 03/03/1961
crearLibro 1935 "El extranjero" "Albert" "Camus" "Aventura" 100
agregarLibroBiblioteca
crearFecha 03/03/1961
crearLibro 1936 "La insoportable levedad del ser" "Milan" "Kundera" "Romantico" 13
agregarLibroBiblioteca
obtenerLibrosDeGenero 18
liberarBiblioteca
Fin
