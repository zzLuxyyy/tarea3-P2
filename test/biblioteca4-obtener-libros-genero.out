1>Biblioteca creada exitosamente.
2>Genero agregado a la biblioteca3>Genero agregado a la biblioteca4>Genero agregado a la biblioteca5>Genero agregado a la biblioteca6>Genero agregado a la biblioteca7>Genero agregado a la biblioteca8>9>10>Libro de ISBN 1984 agregado a la biblioteca.
11>12>13>Libro de ISBN 1932 agregado a la biblioteca.
14>15>16>Libro de ISBN 1933 agregado a la biblioteca.
17>18>19>Libro de ISBN 1934 agregado a la biblioteca.
20>21>22>Libro de ISBN 1935 agregado a la biblioteca.
23>24>25>Libro de ISBN 1936 agregado a la biblioteca.
26>Libro 1933
Título: El Amor en los tiempos de colera
Autor: Gabriel Garcia Marquez
Descripción: Romantico
Género: 18
Fecha de edición: 2/2/1960
Libro 1934
Título: Harry Potter
Autor: JK Rowling
Descripción: Aventura
Género: 18
Fecha de edición: 3/3/1961
Libro 1935
Título: El extranjero
Autor: Albert Camus
Descripción: Aventura
Género: 100
Fecha de edición: 3/3/1961
Libro 1936
Título: La insoportable levedad del ser
Autor: Milan Kundera
Descripción: Romantico
Género: 13
Fecha de edición: 3/3/1961
27>Biblioteca liberada exitosamente.
28>Fin.
